### PR TITLE
Removed not used projects

### PR DESCRIPTION
**Pull request by laidaoyu**
- removed the following projects:
/tools/css-api
/tools/import
/services/net/capture
/services/net/clip
/services/net/command
/services/filecopy
- updated launch.json file

### CODE REVIEW

**Review by Fosol — APPROVED** [review]
There are a few other folders that can be cleaned up too.

`/db/azure-storage`
`/db/kafka/cat`
`/db/kafka/connect`
`/db/kafka/ksqldb`
`/db/kafka/rest-proxy`
`/db/kafka/schema-registry`
`/db/kafka/zookeeper`

**Comment by laidaoyu** [comment]
> There are a few other folders that can be cleaned up too.
> 
> `/db/azure-storage` `/db/kafka/cat` `/db/kafka/connect` `/db/kafka/ksqldb` `/db/kafka/rest-proxy` `/db/kafka/schema-registry` `/db/kafka/zookeeper`

ok I will clean it up. thanks!

File: .vscode/launch.json [hunk 1]
```diff
@@ -45,48 +29,14 @@
       "request": "launch",
       "preLaunchTask": "build-syndication",
       // If you have changed target frameworks, make sure to update the program path.
-      "program": "${workspaceFolder}/services/net/syndication/bin/Debug/net7.0/TNO.Services.Syndication.dll",
+      "program": "${workspaceFolder}/services/net/syndication/bin/Debug/net9.0/TNO.Services.Syndication.dll",
```
[the remark this answers is not in the document]
Comment (laidaoyu) [reply .vscode/launch.json:32]: thanks!

**Comment by laidaoyu** [comment]
> There are a few other folders that can be cleaned up too.
> 
> `/db/azure-storage` `/db/kafka/cat` `/db/kafka/connect` `/db/kafka/ksqldb` `/db/kafka/rest-proxy` `/db/kafka/schema-registry` `/db/kafka/zookeeper`

I have made a new commit for cleaning up these files. Please review. thanks!

**Review by nehalaggarwal-bcgov — APPROVED** [review]
Looks good

**Review by Fosol — APPROVED** [review]

**Pull request merged**